schemas/deployments-schema.yaml

```diff
@@ -142,9 +138,6 @@ properties:
       privacy_loss:
         type: object
         additionalProperties: False
-        required:
-          - privacy_unit
```

Comment:
This one seems worthy of debate or discussion.

Reply:
From my view, and from the discussion we all had previously, "Privacy Loss" is not defined, unless all these fields are here.  We can leave them empty, or highlight that they are unmeasured, but privacy loss is defined with all these.

Reply:
Discussion on 7/31 was to remove this as required, but lean on human review to get these up for deployments.